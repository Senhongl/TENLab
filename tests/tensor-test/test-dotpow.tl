a = [[1.1,2.0],[3.5,4.2]] .^ 2;
print(a);