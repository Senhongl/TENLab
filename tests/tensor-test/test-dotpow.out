  1.2100   4.0000
 12.2500  17.6400
[ CPUDoubleType{2,2} ]
